cronner.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 // Version is the program's version string
-const Version = "0.2.5"
+const Version = "0.2.6"
```

Comment:
Because you're changing the behavior of which output streams are being used, this is not a minor version bump. You're effectively breaking backwards compatibility. Following SemVer, you should probably increase the version to `0.3.0`.